packages/experiments-realm/crm/contact.gts

```diff
@@ -143,12 +164,14 @@ class ViewSocialLinksTemplate extends Component<typeof SocialLinksField> {
         gap: var(--boxel-sp-xxs);
       }
       .social-link {
-        --boxel-icon-button-width: var(--boxel-icon-med);
-        --boxel-icon-button-height: var(--boxel-icon-med);
+        --boxel-icon-button-width: var(--boxel-icon-lg);
+        --boxel-icon-button-height: var(--boxel-icon-lg);
         border: 1px solid var(--boxel-300);
+        border-radius: 5px;
       }
       .social-link:hover {
-        border-color: var(--boxel-400);
+        border-color: var(--boxel-500);
+        background-color: var(--boxel-100);
         cursor: pointer;
```

Comment:
The styling for social media links should be in URLField field def. The version with just the icon might be the atom template. Confirm it with design team. 
I think this should be:`@field socialLinks = **containsMany**(URLField);` based on what the Linear ticket description.